alerta/utils/api.py

```diff
@@ -87,16 +87,16 @@ def process_alert(alert: Alert) -> Alert:
     return alert
 
 
-def process_action(alert: Alert, action: str, text: str) -> Tuple[Alert, str, str]:
-
+def process_action(alert: Alert, action: str, text: str, time_out: int) -> Tuple[Alert, str, str, int]:
+    #named variable time_out so doesnt conflict with kwargs name
     wanted_plugins, wanted_config = plugins.routing(alert)
 
     updated = None
     for plugin in wanted_plugins:
         if alert.is_suppressed:
             break
         try:
-            updated = plugin.take_action(alert, action, text, config=wanted_config)
+            updated = plugin.take_action(alert, action, text, config=wanted_config, timeout=time_out)
```

Comment:
Can you put "timeout" after "text" and before "config" ? The ordering is more consistent that way.

Reply:
done.